README.md

```diff
@@ -29,8 +31,7 @@ further. In this example, we will be focusing on a SYNC test using the Jest fram
 Import the testing library modules in your tests
 ```typescript
 //__tests__/example.test.ts 
-import * as testlib from '../src/validate.ts'
-import * as testreqs from '../src/requests.ts'
+import { testlib, testreqs } from './src'
```

Comment:
A developer should depend on this library as a module that lives in their `node_modules/` directory (even if we don't have it on NPM just yet). So this would be something like
```typescript
import { testlib, testreqs } from 'smart-home-testing'
```

Reply:
Got it!